src/bci_build/package/apache_tomcat.py

```diff
@@ -6,18 +6,17 @@
 from bci_build.package import DOCKERFILE_RUN
 from bci_build.package import OsContainer
 from bci_build.package import OsVersion
-from bci_build.package import ParseVersion
 from bci_build.package import Replacement
 from bci_build.package import _build_tag_prefix
 
 from .appcollection import ApplicationCollectionContainer
 
-_TOMCAT_VERSIONS: list[int] = [9, 10]
-assert _TOMCAT_VERSIONS == sorted(_TOMCAT_VERSIONS)
+# last version needs to be the newest
+_TOMCAT_VERSIONS: list[str] = ["9", "10.1"]
 
 
 def _get_sac_supported_until(
-    os_version: OsVersion, tomcat_major: int, jre_major: int
+    os_version: OsVersion, tomcat_major: str, jre_major: int
```

Comment:
nit: this is no longer only the major version (needs more adjustment down the line too)
```suggestion
    os_version: OsVersion, tomcat_ver: str, jre_major: int
```

Reply:
all references renamed.